Update the style guide to mention attributes (#161)
bosagora/agora: doc/Style.md

Templated functions are written as `void foobar (T) (T arg)`.
This makes functions easy to `grep` for.

### Attributes (@trusted, @safe, @nogc, pure, etc)

Prefer adding `@safe`, `@nogc`, and `pure` atttributes to functions when it's

> possible. If one or more statements inside of the functions call a non-safe or

Reviewer: ```suggestion
possible. If one or more statements inside a non-trivial function call a non-safe or
```

Templated functions are written as `void foobar (T) (T arg)`.
This makes functions easy to `grep` for.

### Attributes (@trusted, @safe, @nogc, pure, etc)

Prefer adding `@safe`, `@nogc`, and `pure` atttributes to functions when it's

> possible. If one or more statements inside of a non-trivial function calla a

Reviewer: ```suggestion
possible. If one or more statements inside of a non-trivial function call a
```
